rasa_core/policies/embedding_policy.py

```diff
@@ -1404,6 +1404,10 @@ def persist(self, path: Text) -> None:
         with io.open(dump_path, 'wb') as f:
             pickle.dump(self.encoded_all_actions, f)
 
+        dump_config_path = os.path.join(path, file_name + ".config.pkl")
```

Comment:
since it is only tf_config let's name it
```suggestion
         dump_config_path = os.path.join(path, file_name + ".tf_config.pkl")
```